readme.md

```diff
@@ -139,6 +139,7 @@
 - [PureScript](https://github.com/passy/awesome-purescript#readme)
 - [Go](https://github.com/avelino/awesome-go#readme)
 - [Scala](https://github.com/lauris/awesome-scala#readme)
+	- [Scala Native](https://github.com/tindzk/awesome-scala-native#readme)
```

Comment:
You need a description here about what Scala Native is (NOT your list). See other descriptions.

Reply:
Thanks. I expanded the description with more information on Scala Native.

Reply:
I think you misunderstood me. You need a short description exactly **here** (above). This is not about your list.